orttraining/orttraining/python/training/orttrainer.py

```diff
@@ -0,0 +1,38 @@
+from .optim import _OptimizerConfig
+
+class TrainStepInfo(object):
+    r"""Private class used to store runtime information from current train step.
+
+    After every train step, :py:meth:`ORTTrainer.train_step` updates the internal instance of
+    :py:class:`.TrainStepInfo` residing on :py:class:`.ORTTrainer` with relevant information
+    from the forward pass.
+
+    This class shouldn't be accessed directly by the user, unless they really know what they are doing.
```

Comment:
It is good to mention this. Maybe just add a few words: "This class is intended for use in ORTTrainer, and should not be accessed directly..."

Reply:
The next sentence is something like ":py:class:`.ORTTrainer` passes it to relevant class methods automatically, such as :py:method:`._LRScheduler.get_lr` or :py:class:`.LossScaler.update`"